app/models/item.rb

```diff
@@ -9,6 +9,10 @@ def price
     sprintf('%.2f', read_attribute(:price))
   end
 
+  def self.search(query)
+    where { title =~ "#{query}%" }
```

Comment:
Minor note so you are aware - this will return only items **starting** with the query. To have the query return if it matches anything in the search result, you'd need to do `"%#{query}%"`, but it will be a bit slower.